src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -130,11 +131,11 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
 
                             using (var fs = new FileStream(output.FullName, FileMode.Create, FileAccess.Write))
                             {
-                                Console.Out.WriteLine($"Process     : {process.MainModule.FileName}");
-                                Console.Out.WriteLine($"Output File : {fs.Name}");
-                                Console.Out.WriteLine($"\tSession Id: 0x{sessionId:X16}");
-                                if (duration != null)
-                                    Console.WriteLine($"Tracing for {duration.ToString(@"dd\:hh\:mm\:ss")}");
+                                Console.Out.WriteLine($"Process        : {process.MainModule.FileName}");
+                                Console.Out.WriteLine($"Output File    : {fs.Name}");
+                                if (shouldStopAfterDuration)
+                                    Console.Out.WriteLine($"Trace Duration : {duration.ToString(@"dd\:hh\:mm\:ss")}");
+                                Console.Out.WriteLine($"\tSession Id: 0x{sessionId:X16}\n");
```

Comment:
nit: This changes the order of SessionId and Trace Duration printed on the console. Is this intentional?

Reply:
It was; I should have mentioned that in the commit message or PR comment.  I thought it looked odd for the output to go:
```log
key : value
key : value
    key : value
string with no colon char
```

So I aligned it with the other output.